privacyidea/lib/resolvers/SQLIdResolver.py

```diff
@@ -395,7 +399,7 @@ def getUserId(self, LoginName):
                     raise Exception("More than one user with loginname"
                                     " %s found!" % LoginName)
                 user = self._get_user_from_mapped_object(r)
-                userid = user["id"]
+                userid = str(user["id"])
```

Comment:
Also here we might have the same problem, I guess ``six.text_type would be safer``

Reply:
Fixed in 61743ea